src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -732,7 +732,7 @@ public IActionResult CreateNewAccount([FromBody]GetUnusedAccountModel request)
         /// Gets an unused address.
         /// </summary>
         /// <returns>The last created and unused address or creates a new address (in Base58 format).</returns>
-        [Route("address")]
+        [Route("newaddress")]
```

Comment:
Won't `[Route("unusedaddress")]` be a better name perhaps?

Reply:
yeah adds more clarity as to how we select the new address.
same for below comment.
Thanks :-)